src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -92,8 +95,18 @@ extends AbstractMap[Key, Value]
     deleted = 0
   }
 
+  /** Return the index of the first slot in the hash table (in probe order)
+    * that either is empty, or is or was last occupied by the given key.
+    */
   private[this] def findIndex(key: Key) : Int = findIndex(key, hashOf(key))
 
+  /** Return the index of the first slot in the hash table (in probe order)
+    * that either is empty, or is or was last occupied by the given key.
```

Comment:
Yes, there's undocumented semantics in the internal structure. I'm not trying to document everything (like it should have been when created), because I have other changes in mind that will remove said structure.